src/contracts/calle.ts

```diff
@@ -0,0 +1,31 @@
+import { assert } from "console"
+import { ByteString, SigHash, SmartContract, Utils, hash256, int2ByteString, method, prop } from "scrypt-ts"
+

```

Comment:
For each contract, let us add a link to the article, if any, as in https://github.com/sCrypt-Inc/boilerplate/blob/b649ab516abb5eda8c2e442c1a752252abe02446/src/contracts/accumulatorMultiSig.ts#L14

https://xiaohuiliu.medium.com/inter-contract-call-on-bitcoin-f51869c08be